pandas/tseries/period.py

```diff
@@ -325,7 +322,6 @@ def _shallow_copy_with_infer(self, values=None, **kwargs):
 
     def _shallow_copy(self, values=None, **kwargs):
         if kwargs.get('freq') is None:
```

Comment:
I would explicitly list ``freq=None`` I think in the signature (then pass it directly) as its so crucial to the PeriodIndex nice to be noted